src/components/dashboard/DownloadModal.tsx

```
@@ -0,0 +1,158 @@
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        {/* Content Preview Section */}
+        <div className="space-y-3">
+          <h4 className="text-sm font-semibold text-white/80">Content Preview</h4>
+          <div className="bg-[#110D18]/40 border border-[#1D1D1C] rounded-lg p-4 text-sm text-white/60 max-h-40 overflow-y-auto">
+            <p className="leading-relaxed">
+              This document contains the student's response to the Research & Write
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)